tests/framework/CodeInterfaceTests/CobraTF/CobraTFinterfaceTest5/testDummyStep_Single/1/vuq_param.txt

```diff
@@ -0,0 +1 @@
+k_db_1 = 1000.0
```

Comment:
Seems to me, you do not need this file. This file will generated by raven, right?